### ISSUE TITLE
[MAS1.3.1] [Screen Reader-Live Chat] Screen reader is not reading the Time information when the bot or user send the message.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1759]
**Actual Result:**
Screen reader is not reading the Time information when the bot or user send the message 
Screen reader is reading as User said “how are “button 
Role is defined for bot echo and user messages text.

**Expected Result:**
Screen reader should read the Time information when the bot or user send the message 
Screen reader should read as “A minute ago User said how are “
Role(Button) should be removed for user and bot echo messages texts 

**Note:** Issue is repro’ing with NVDA. Please refer the attachment as <MAS1.3.1_Live chat_NVDA.wmv>
[MAS1.3.1_Live chat_NVDA.zip](https://github.com/microsoft/BotFramework-Emulator/files/3545896/MAS1.3.1_Live.chat_NVDA.zip)

Same issue observed in the Mac also. Please refer the attachment for more information (Screen reader is not reading the Time information_MAC) 

**Repro Steps:**
Turn on the narrator 
1. Open the Bot Framework V4 Emulator.
2.Welcome tab screen gets opened
3. Navigate to the Open a bot and select it 
4.Open bot dialog gets opened and connect the pen bot 
5.Live chat screen gets opened 
6.Naviagte to the "How are " text message
7. Observe the issue

**User Impact:**	
Users who rely on AT will be affected if the screen reader is not reading the descriptive information then the screen reader user will skip the feature 

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator
Screen reader: Narrator 

OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

**MAS Reference Link:**
[Sharepoint Link](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})
[MAS1.3.1_Live chat.zip](https://github.com/microsoft/BotFramework-Emulator/files/3524912/MAS1.3.1_Live.chat.zip)
![MAS1 3 1_Live chat](https://user-images.githubusercontent.com/53518994/63425422-1ddbe200-c42e-11e9-9354-ac15cb375e41.png)

[Screen reader is not reading the Time information_Mac.zip](https://github.com/microsoft/BotFramework-Emulator/files/3691103/Screen.reader.is.not.reading.the.Time.information_Mac.zip)

**Comment by corinagum** [comment]
I'm fairly confident this will be fixed when we bump Web Chat in Emulator. https://github.com/microsoft/BotFramework-WebChat/pull/2278

**Comment by corinagum** [comment]
Waiting for this one until towards the end of our audit since the bump to React 16.9+ will cause merge conflicts with PRs.

**Comment by Kumar2608** [comment]
Verified in latest bot emulator version 4.5.2-82861 with NVDA(2019.2) Issue partially fixed Hence we are reactivating this bug.
Screen reader reading the date information but time info is not narrating .
Please refer the attachment:
[1759.zip](https://github.com/microsoft/BotFramework-Emulator/files/3728492/1759.zip)
[Bug 1759.zip](https://github.com/microsoft/BotFramework-Emulator/files/3728497/Bug.1759.zip)

**Note:** Issue still repro'ing for both Win32 and MAC 
For MAC version-4.5.2-82590 and Mac version-10.15(Beta)

**Comment by corinagum** [comment]
@Kumar2608 the first video is reading the time stamp (Windows) as well as the date. It looks like this bug is only repro-ing on Mac. Can I get confirmation?

**Comment by corinagum** [comment]
@Kumar2608 I have checked this both on Windows and Mac and both are reading both the date and time. I have a recording of the Mac repro, but unfortunately I am having WiFi problems and I am unable to upload the video yet. I will update this thread once I get that video up.

**Comment by corinagum** [comment]
@Kumar2608, here's that video. My apologies for the bad quality.

[timestamp.zip](https://github.com/microsoft/BotFramework-Emulator/files/3736537/timestamp.zip)

**Comment by Kumar2608** [comment]
As per dev issue got fixed hence we are closing this bug.
